qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -99,9 +97,9 @@ def _direction_condition(property_set):
     if coupling_map:
         pm0.append(_swap_check)
         pm0.append(_swap, condition=_swap_condition)
-        # pm0.append(_direction_check)  # TODO
-        pm0.append(_direction, condition=_direction_condition)
+        pm0.append(CXDirection(coupling_map), condition=_direction_condition)
     pm0.append(_reset)
-    pm0.append(_direction)
```

Comment:
Actually i think the fix is just to remove this. This is included by mistake (it doesn't do anything new, it already gets called above).
This same bug exists in level1 as well. 
Can you remove this line from both level 0 and level 1 please?